cfgov/v1/migrations/0142_migrate_pas_link_data_to_pagechooserblock.py

```diff
@@ -15,6 +15,18 @@
 
 
 def forward_mapper(page_or_revision, data):
+    # Set the default disclaimer_page_id, which corresponds to
+    # the Generic Email Sign-Up Privacy Act Statement.
+    disclaimer_page_id = 1189
+    data['disclaimer_page'] = disclaimer_page_id
+
+    if not data.get('form_field'):
+        return data
+    elif not data['form_field']:
```

Comment:
This is redundant, but does no harm.

Reply:
I don't _think_ it is, from what I experienced writing the tests. The first intends to test whether `data['form_field']` is there at all, the second intends to test whether the `data['form_field']` list is empty.

Reply:
If data['form_field'] is empty (e.g. an empty string), it will be handled in the first conditional, and never reach the second.

Reply:
Yeah, you're right, taking the middle one out still passes all the tests. Update pushed.